src/ServiceControl.Monitoring/ServiceControl.Monitoring.csproj

```diff
@@ -12,11 +12,26 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>ServiceControl.Monitoring</RootNamespace>
     <AssemblyName>ServiceControl.Monitoring</AssemblyName>
-    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
```

Comment:
If this targets anything higher that 4.5.2 then we need to update the PI and the MSI for ServiceControl.  The PI installs 4.5.2 if it is not present and the SC MSI install has checks in it to determine whether the required framework is there